mongo-fixes/delete-published-collection-id/delete-published-collection-id.js

```diff
@@ -0,0 +1,73 @@
+// delete-published-collection-id.js
+//
+// if a dataset doc has:
+//     next.state:"published" and non-blank    next.collection_id ---> delete latter
+//  current.state:"published" and non-blank current.collection_id ---> delete latter
+
+// collection
+ds_collection = 'datasets'
+
+// sub-document parts
+ds_subdocs = ['next', 'current']
+
+// do a find first, show what would be changed
+show_find         = true
```

Comment:
Lines 14 and 15, would it better if these were flags passed into script? No point having an if statements in for loop if these are hardcoded to be always true?

Reply:
I've attempted some form of options, hope you love it. 😛